infra/build/functions/build_project.py

```diff
@@ -263,13 +279,13 @@ def get_build_steps(  # pylint: disable=too-many-locals, too-many-statements, to
     return []
 
   timestamp = get_datetime_now().strftime('%Y%m%d%H%M')
-
   build_steps = build_lib.project_image_steps(
       project.name,
       project.image,
       project.fuzzing_language,
       branch=config.branch,
-      test_image_suffix=config.test_image_suffix)
+      test_image_suffix=config.test_image_suffix,
+      build_arm=has_arm_build(project))
```

Comment:
Can we just pass `project.architectures` instead?

Reply:
Done.